### ISSUE TITLE
option to adjust dimensions [was: output png extremely small]

### ISSUE THREAD

**Issue by iblislin** [pkra/mathjax-node-svg2png#3]
I guess this causes the output png sooo small: https://github.com/pkra/mathjax-node-svg2png/blob/master/lib/main.js#L55

Here is a quick hack, hope this help
```diff
--- main.js.orig        2017-04-04 17:17:19.159440000 +0800
+++ main.js     2017-04-04 17:18:20.911237000 +0800
@@ -45,15 +45,14 @@
     if (data.png) data.svg = true;
     mathjax.typeset(data, function(result) {
         if (result.error) callback(result);
-        if (data.png) convert(result, callback);
+        if (data.png) convert(data, result, callback);
     });
 };

-var convert = function(result, callback) {
+var convert = function(data, result, callback) {
     var sourceBuffer = new Buffer(result.svg, "utf-8");
     var returnBuffer = svg2png.sync(sourceBuffer, {
-        width: result.width,
-        height: result.height
+        width: data.width,
     });      
     result.png = "data:image/png;base64," + returnBuffer.toString('base64');
     callback(result);
```

**Comment by pkra** [comment]
Thanks for the feature request. The old mathjax-node batik integration did have a "DPI" option.

I would approach it somewhat differently once https://github.com/mathjax/MathJax-node/issues/303 is fixed. I'd also prefer a more specific name.

Would a scaling factor or "DPI" option be more useful?

**Comment by iblislin** [comment]
> Would a scaling factor or "DPI" option be more useful?

sure. 👍

**Comment by pkra** [comment]
I finally got around to this. Let me know what you think about #5

**Comment by pkra** [comment]
Fixed via #5
